src/query/ee/tests/it/aggregating_index/index_scan.rs

```diff
@@ -614,21 +691,29 @@ async fn test_index_scan_agg_args_are_expression_impl(format: &str) -> Result<()
 
     // Create index
     let index_name = "index1";
-
+    fixture
+        .ctx()
+        .evict_table_from_cache(CATALOG, DATABASE, "t")?;
     execute_sql(
         fixture.ctx(),
         &format!("CREATE AGGREGATING INDEX {index_name} AS SELECT SUBSTRING(a, 1, 1) as s, sum(length(a)), min(a) from t GROUP BY s"),
     )
         .await?;
 
     // Refresh Index
+    fixture
+        .ctx()
+        .evict_table_from_cache(CATALOG, DATABASE, "t")?;
     execute_sql(
         fixture.ctx(),
         &format!("REFRESH AGGREGATING INDEX {index_name}"),
     )
     .await?;
 
     // Query with index
+    fixture
+        .ctx()
+        .evict_table_from_cache(CATALOG, DATABASE, "t")?;
```

Comment:
does the correctness of testing depend on those invocations of `evict_table_from_cache`?

Reply:
yes, please see the summary session 2 of this pr. In real sql execution, every sql has its own ctx, but for ut test, they will execute many ctx but have the same ctx, so the later pipeline will get the old snapshot info which is from previous ones. cc @dantengsky

Reply:
oh, I see.

seems the related test case should be adjusted (by using new query context ?), let's keep the changes as it is, and fix them in another PR.

Reply:
turns out, an issue is tracking this 

https://github.com/datafuselabs/databend/issues/11887